front/cypress/e2e/admin/pages_and_menu/update_homepage_hero_banner_content.cy.ts

```diff
@@ -77,6 +77,30 @@ describe('Admin: update Hero Banner content', () => {
     );
   });
 
+  it('uploads and crops banner image', () => {
+    cy.intercept('PATCH', '**/home_page').as('saveHomePage');
+    cy.setAdminLoginCookie();
+    cy.visit('admin/pages-menu/homepage/homepage-banner/');
+    cy.acceptCookies();
```

Comment:
Is this needed given we set the cookie line 82?

Reply:
I added this line because the test didn't work. `setAdminLoginCookie` sets only `cl2_jwt`.

Reply:
Didn't check the code, but maybe this could be added to setAdminLoginCookie? Or that function should be renamed to avoid confusion? :)

Reply:
I can do it.

Reply:
It seems `setAdminLoginCookie` does exactly what it says: sets the login cookie and nothing more (consent is stored in another cookie). So, the name is correct, and if this function clicks some buttons to accept cookies (e.g., by calling `acceptCookies`), the name becomes incorrect.

There are still some options, but none of them is good enough.

1. I tried setting consent cookies in `setAdminLoginCookie` without calling `acceptCookies`, but because of the tricky `js-cookie` [converter](https://github.com/js-cookie/js-cookie/blob/main/src/converter.mjs), it looks ugly (no type checks, very brittle):
```ts
export function setConsentCookie() {
  cy.setCookie(
    'cl2_consent',
    '{%22analytics%22:true%2C%22advertising%22:true%2C%22functional%22:true%2C%22savedChoices%22:{%22google_tag_manager%22:true%2C%22matomo%22:true%2C%22google_analytics%22:true%2C%22intercom%22:true%2C%22segment%22:true}}'
  );
}
```

2. Renaming `setAdminLoginCookie` and calling `acceptCookies` in there would require some refactoring (remove all `acceptCookies` which are not needed), so we could add a new function. But surprisingly, it doesn't work.

```ts
export function setAdminLoginCookieAndAcceptCookies() {
  cy.setAdminLoginCookie();
  cy.acceptCookies();
}
```
So, it seems the best option is to leave it as it is now.